docs/overview/header.md

```diff
@@ -5,3 +5,74 @@ to the top of the viewport even when the main content area is scrolled up and do
 In order to create a header you have to create a CDI bean that implement org.uberfire.client.workbench.Header interface.
 
 ![header](header.png)
+
+Let us add a simple Composite that adds 3 labels. 
+
+In package org.uberfire.client.navbar we can create 
+
+- HeaderView.java
+
+```
+package org.uberfire.client.navbar;
+
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.HorizontalPanel;
+import com.google.gwt.user.client.ui.Label;
+
+public class HeaderView extends Composite {
+
+    public HeaderView() {
```

Comment:
Errai UI docs: https://docs.jboss.org/author/display/ERRAI/Errai+UI?_sscc=t
We should use MVP patterns in all of our widgets.

Here is a example https://github.com/uberfire/uberfire-tutorial/blob/master/uftasks/uftasks-showcase/uftasks-webapp/src/main/java/org/uberfire/client/screens/ProjectsPresenter.java (you don't need a uf screen, just the presenter view part).

We are trying to avoid as much as we can use gwt widgets in our code base.